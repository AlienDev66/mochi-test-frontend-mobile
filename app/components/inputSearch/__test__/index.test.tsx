import React from 'react';
import renderer from 'react-test-renderer';
import InputSearch from '..';

describe('<InputSearch />', () => {
    const tree = renderer
        .create(<InputSearch onPress={() => {}} onChangeText={() => {}} />)
        .toJSON();
    it('has 2 child', () => {
        if (tree) expect(tree.children.length).toBe(2);
    });
    it('has 3 child', () => {
        if (tree) expect(tree).toMatchSnapshot();
    });
});
