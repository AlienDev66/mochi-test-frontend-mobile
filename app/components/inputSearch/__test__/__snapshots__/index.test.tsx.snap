// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputSearch /> has 3 child 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "borderColor": "#acb7c5",
        "borderRadius": 5,
        "borderStyle": "solid",
        "borderWidth": 1,
        "flexDirection": "row",
        "height": 50,
        "justifyContent": "space-between",
        "marginLeft": 25,
        "marginRight": 25,
        "width": 300,
      },
    ]
  }
>
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    placeholder="Type a user name here"
    rejectResponderTermination={true}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "color": "black",
          "height": 50,
          "paddingBottom": 10,
          "paddingLeft": 10,
          "paddingRight": 10,
          "paddingTop": 10,
          "width": "85%",
        },
      ]
    }
    underlineColorAndroid="transparent"
  />
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#c200b9",
        "borderRadius": 2,
        "height": 32,
        "justifyContent": "center",
        "marginRight": 5,
        "opacity": 1,
        "width": 32,
      }
    }
  >
    <Text />
  </View>
</View>
`;
